go.mod

```diff
@@ -144,3 +144,5 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gotest.tools/v3 v3.1.0 // indirect
 )
+
+replace github.com/chainguard-dev/go-apk => github.com/thesayyn/go-apk v0.0.0-20231011191853-e0653a72ad44
```

Comment:
We should remove this replace before merging.

Reply:
done